application/controllers/User.php

```diff
@@ -798,7 +868,23 @@ function login() {
 
 				);
 				$this->input->set_cookie($cookie);
+
+				// Create a keep_login cookie
+				if ($this->input->post('keep_login') == '1') {
+
+					$encrypted_string = $this->user_model->keep_cookie_hash($data['user']->user_id);
+
+					$cookie = array(
+						'name'   => 'keep_login',
+						'value'  => $encrypted_string,
+						'expire' => '2592000',  // 30 days
+						'secure' => TRUE
```

Comment:
This one will perhaps fail, if WL runs in a non-TLS-Environment.

Reply:
not sure. the 'keep logged in' checkbox is hidden if the page is called without https

Reply:
fine for me as well. So no "remember me" for HTTP-only users :)